### PR TITLE
Current content of cheatsheet structured

### PR DESCRIPTION
**Pull request by Legersky**
- tables for cross-referencing
- code of sample definition

### CODE REVIEW

**Comment by Legersky** [comment]
I have realized that we have to distinguish in the cheatsheet Myst and reST referencing.

**Comment by Legersky** [comment]
- reST added
 - math in definition

File: doc/development/cheatsheet.md [hunk 1]
```diff
@@ -1,33 +1,117 @@
...
 
-{meth}`~pyrigi.framework.Framework.delete_edge`
+::::{tab-item} reST  (docstrings)
+:sync: rest
 
-{meth}`.Framework.delete_edge`
+:::{csv-table}
+`` :prf:ref:`def-framework` `` , {prf:ref}`def-framework`
+`` :prf:ref:`Framework <def-framework>` `` , {prf:ref}`Framework <def-framework>`
+`` :cite:p:`Geiringer1927` `` , {cite:p}`Geiringer1927`
```
Comment (matthiashimmelmann) [inline doc/development/cheatsheet.md:26]: Can you do an example for multiple citations?

**Review by matthiashimmelmann — CHANGES_REQUESTED** [review]
Should the cheat sheet also entail things like 
How to set up the documentation-compilation? 
How to run pytest (with and without slow tests)?
Or is that rather part of the handbook?

File: doc/development/cheatsheet.md (hunk 1 above)
Comment (Legersky) [reply doc/development/cheatsheet.md]: Done

**Comment by Legersky** [comment]
> Should the cheat sheet also entail things like How to set up the documentation-compilation? How to run pytest (with and without slow tests)? Or is that rather part of the handbook?

I would say this is another part of the development guid.

**Review by matthiashimmelmann — APPROVED** [review]

**Pull request merged**